src/providers/baseProviderWrapper.ts

```diff
@@ -1,16 +0,0 @@
-/* eslint-disable class-methods-use-this, @typescript-eslint/no-empty-function */
-
-import { ProviderTypes } from '../types/provider';
-
-export abstract class ProviderWrapper {
-  type?: ProviderTypes;
-  provider: any;
-
-  constructor(provider) {
-    this.provider = provider;
-  }
-
-  abstract getAllData(address: string, keys: string[]): Promise<any>;
-  abstract getData(address: string, keyHash: string): Promise<any>;
-  abstract getOwner(address: string): Promise<any>;
-}
```

Comment:
@rryter what was the reason for removing the base class and not using extends?

I'd like to introduce it again so all our providers have same functions/signatures.
Atm, `getAllData` of Graph provider has a different signature than web3|ethereum providers.

![image](https://user-images.githubusercontent.com/477945/139530826-9b8ad747-c63e-4871-a911-59dd8f6a36bc.png)